src/capital/capital_models.jl

```diff
@@ -2,4 +2,6 @@ abstract type CapitalCostModel end
 
 struct DiscountedCashFlow <: CapitalCostModel
     discount_rate::Float64
+    base_year::Dates.Year
+    investment_years::Vector{NTuple{2, Dates.Date}}
```

Comment:
is this duplicate from the chronology?

Reply:
the users must specify the tuple of years for the investment and then we construct the mapping, so its the minimum needed from the user. We are not requiring the user to do the time mapping, but rather specify in the CapitalCostModel the investment periods.